.storybook/components/Docs/Documentation.module.css

```diff
@@ -0,0 +1,105 @@
+.documentation {
```

Comment:
Another possible name is `.markdown` and the component is `MarkDownWrapper` or something... 🤔?

Reply:
I'm down for `MarkDownWrapper` and `.markdown`